README.md

```diff
@@ -1265,7 +1276,37 @@ Example result:
 
 ```
 {'defaultAggregation': 'AVG', 'metric': 'nvt'}
+
 ```
+
+## Extras
+
+### Backtest
+
+TBD
+
+## Development
+
+It is recommended to use [pipenv](https://github.com/pypa/pipenv) for managing your local environment.
+
+Setup project:
+
+```
+pipenv install
```

Comment:
I generally prefer `asdf` as version manager because it can be used also for elixir and other language envs we have. But I guess we want be able to have separate dependancies that way :?

Reply:
`pipenv` is considered the pythonic way, it is developed under [Python Packaging Authority](https://github.com/pypa).